File: data/src/main/java/com/losingtimeapps/data/model/RepositoryResponse.java

package com.losingtimeapps.data.model;

import java.util.List;
import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class RepositoryResponse {

    @SerializedName("incomplete_results")
    private Boolean mIncompleteResults;
    @SerializedName("items")
    private List<Item> mItems;
    @SerializedName("total_count")
    private Long mTotalCount;

    public Boolean getIncompleteResults() {
        return mIncompleteResults;
    }

    public void setIncompleteResults(Boolean incompleteResults) {
        mIncompleteResults = incompleteResults;
    }

    public List<Item> getItems() {
        return mItems;
    }

    public void setItems(List<Item> items) {
        mItems = items;
    }

    public Long getTotalCount() {
        return mTotalCount;
    }

    public void setTotalCount(Long totalCount) {
        mTotalCount = totalCount;
    }

}

File: app/src/main/java/com/accenture/desafioandroid/mvvm/model/RepositoryResponse.kt
package com.accenture.desafioandroid.mvvm.model

import com.accenture.desafioandroid.mvvm.model.Item
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RepositoryResponse {

    @SerializedName("total_count")
    @Expose
    var totalCount: Int? = null
    @SerializedName("incomplete_results")
    @Expose
    var incompleteResults: Boolean? = null
    @SerializedName("items")
    @Expose
    var items: MutableList<Item?>? = null

}